src/ftp.rs

```diff
@@ -20,27 +24,125 @@ lazy_static! {
     static ref SIZE_RE: Regex = Regex::new(r"\s+(\d+)\s*$").unwrap();
 }
 
+#[cfg(feature = "secure")]
+lazy_static! {
+    // Shared SSL context
+    static ref SSL_CONTEXT: SslContext = match SslContext::new(SslMethod::Sslv23) {
```

Comment:
(Although I'm not sure whether this will set the method to TLS v1.x or to TLS v1.0. If the latter is true, it's a bad choice.)